src/sentry/templatetags/sentry_assets.py

```diff
@@ -5,12 +5,13 @@
 from django.utils.safestring import mark_safe
 
 from sentry import options
-from sentry.utils.assets import get_asset_url
+from sentry.utils.assets import get_asset_url, get_manfifest_url
 from sentry.utils.http import absolute_uri
 
 register = template.Library()
 
 register.simple_tag(get_asset_url, name="asset_url")
```

Comment:
Wouldn't we need to remove all usage of asset_url and get_asset_url? Unless you plan on building both for now, which... would be a lot.

Reply:
We will need to keep `asset_url` as there are files that webpack does not touch, I think ideally we'd have these assets go through webpack too

Reply:
MM I may do a pass to pipe these assets through webpack